releng/org.eclipse.n4js.parent/pom.xml

```diff
@@ -346,12 +340,6 @@ Contributors:
 								<goal>add-source</goal>
 							</goals>
 							<configuration>
-								<sources>
-									<source>src-gen</source>
-								</sources>
-								<sources>
-									<source>emf-gen</source>
-								</sources>
 								<sources>
 									<source>grammar-gen</source>
```

Comment:
it seems inconsistent that `src-gen`, `emf-gen` are removed whereas `grammar-gen` isn't removed.